handlers/api.go

```diff
@@ -2147,7 +2148,25 @@ func ApiGraffitiwall(w http.ResponseWriter, r *http.Request) {
 	if slotQuery < 10000 {
 		slotQuery = 10000
 	}
-	rows, err := db.ReaderDb.Query("SELECT x, y, color, slot, validator FROM graffitiwall WHERE slot <= $1 AND slot >= $2 ORDER BY slot desc", slotQuery, slotQuery-10000)
+
+	startPxl := uint64(0)
+	endPxl := uint64(999)
+
+	startX := math.MinU64(parseUintWithDefault(r.URL.Query().Get("startx"), startPxl), endPxl)
+	startY := math.MinU64(parseUintWithDefault(r.URL.Query().Get("starty"), startPxl), endPxl)
+	endX := math.MinU64(parseUintWithDefault(r.URL.Query().Get("endx"), endPxl), endPxl)
+	endY := math.MinU64(parseUintWithDefault(r.URL.Query().Get("endy"), endPxl), endPxl)
```

Comment:
New parameters have not been added to swagger comment above this function. Please add them. Maybe mention that both values are inclusive (as querying from 50/50 to 50/50 works).

Please think about further required changes for the swagger comment too.

Reply:
Adjusted everything in the swagger comment.